app/scripts/lib/app-start.js

```diff
@@ -589,23 +589,13 @@ define(function (require, exports, module) {
     },
 
     allResourcesReady: function () {
-      // The IFrame cannot use pushState or else a page transition
```

Comment:
I checked Fx 18 even and could not find where this problem manifested. Perhaps old IE?